source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/project-table.scss

```diff
@@ -1,7 +1,82 @@
-.project-actions {
-  color: #4a4a4a;
+.project-table {
 
-  i {
-    opacity: 0.5;
+  color: #515151;
+  font-family: Montserrat;
+  font-size: 13px;
+
+  a {
+    font-family: Montserrat;
+    font-size: 13px;
+    font-weight: bold;
+    color: #4a90e2;
+  }
+
+  .header {
+
+    font-weight: bold;
+    border-bottom: 2px solid #515151;
+    background: transparent;
+
+  }
+
+  .row-header {
+    background-color: #fbfbfb;
+    border: solid 1px #dfdfdf;
+    margin-top: 6px;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    font-weight: bold;
+  }
+
+  .grid {
+
+    .col {
+      width: 32%;
+      padding: .92857143em .78571429em;
+    }
+    .action {
+      width: 4%;
+      padding: .92857143em .78571429em;
+      text-align: center;
+
+      i {
+        margin: 0;
+      }
+    }
+  }
+
+  .products-grid {
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+    box-shadow: 0 1px 0 0 #d8d8d8;
+    background-color: #ffffff;
+    border: solid 1px #d9d9d9;
+    padding-left: 14px;
+    padding-right: 14px;
+
+    .products-header {
+      font-family: Montserrat;
```

Comment:
does the font need to be specified?

Reply:
yes, for some reason it's not inherit from body. Later on we can check why since it's not exactly part of the story